script.js

```diff
@@ -1,269 +1,303 @@
+//    } else if (optionsParent.selectedOptions[0].value === 'time') {
+//        optionsChildOne.innerHTML = timeInnerHTML('first')
+//        optionsChildTwo.innerHTML = timeInnerHTML()
+//    }
+//})
+//
+//// If user selects an option(child) as same as other child, swap them up
+//let prevOpt1 = optionsChildOne.value
+//let prevOpt2 = optionsChildTwo.value
+//optionsChildOne.addEventListener('click', () => {
+//    prevOpt1 = optionsChildOne.value
+//    prevOpt2 = optionsChildTwo.value
+//})
+//
+//optionsChildTwo.addEventListener('click', () => {
+//    prevOpt1 = optionsChildOne.value
+//    prevOpt2 = optionsChildTwo.value
+//})
+//
+//optionsChildOne.addEventListener('change', () => {
+//    resetBoxesValues()
+//    if (optionsChildOne.value === optionsChildTwo.value) {
+//        optionsChildTwo.value = prevOpt1
+//    }
+//})
+//
+//optionsChildTwo.addEventListener('change', () => {
+//    resetBoxesValues()
+//    if (optionsChildTwo.value === optionsChildOne.value) {
+//        optionsChildOne.value = prevOpt2
+//    }
+//})
+//
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+//// Swap Function
+//const swap = () => {
+//    [firstBox.value, secondBox.value] = [secondBox.value, firstBox.value]
+//
+//    let temporary = optionsChildOne.value
+//    optionsChildOne.value = optionsChildTwo.value
+//    optionsChildTwo.value = temporary
+//}
+//
+//swapper.addEventListener('click', swap)
+//
+//// Reset Boxes' values Function
+//function resetBoxesValues() {
+//    firstBox.value = ''
+//    secondBox.value = ''
+//}
+//
+//// checks if box is empty and assign next box also empty string
+//// we need this because when we calculate something with empty string('') it gives number such as '' * 2 = 0 or '' - 2 = -2
+//const checker = (box1, box2) => {
+//    if (box1.value === '') {
+//        box2.value = ''
+//        return true
+//    }
+//}
+//
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+//
+//// Convertion Classes
+//
+//
+//// Length 
+//class Length {
+//    constructor() {
+//        this.kmToMi = (box1, box2) => box1.value = +(box2.value / 1.60934).toFixed(6)
+//        this.miToKm = (box1, box2) => box1.value = +(box2.value * 1.60934).toFixed(6)
+//        this.kmToM = (box1, box2) => box1.value = +(box2.value * 1000).toFixed(6)
+//        this.mToKm = (box1, box2) => box1.value = +(box2.value / 1000).toFixed(6)
+//        this.kmToCm = (box1, box2) => box1.value = +(box2.value * 100000).toFixed(6)
+//        this.cmToKm = (box1, box2) => box1.value = +(box2.value / 100000).toFixed(6)
+//        this.kmToFt = (box1, box2) => box1.value = +(box2.value * 3280.839895).toFixed(6)
+//        this.ftToKm = (box1, box2) => box1.value = +(box2.value / 3280.839895).toFixed(6)
```

Comment:
remove all the commented lines, you can find the previous code in the git history anyway so no need for them